contracts/liquidityMining/KyberRewardLocker.sol

```diff
@@ -221,38 +211,43 @@ contract KyberRewardLocker is IKyberRewardLocker, PermissionAdmin {
   {
     VestingSchedules storage schedules = accountVestingSchedules[msg.sender][token];
     uint256 totalVesting = 0;
-    uint256 totalSlashing = 0;
     for (uint256 i = 0; i < indexes.length; i++) {
       VestingSchedule memory schedule = schedules.data[indexes[i]];
       if (schedule.quantity == 0) {
         continue;
       }
       uint256 vestQuantity = _getVestingQuantity(
         schedule.quantity,
-        schedule.startTime,
-        schedule.endTime
+        schedule.startBlock,
+        schedule.endBlock
       );
       if (vestQuantity == 0) {
         continue;
       }
       totalVesting = totalVesting.add(vestQuantity);
-      totalSlashing = totalSlashing.add(schedule.quantity - vestQuantity);
-      // clear data after vesting
-      schedules.data[i].quantity = 0;
+
+      if (vestQuantity == uint256(schedule.quantity)) {
```

Comment:
should update `schedules.data[indexes[i]]` instead of `schedules.data[i]`

Reply:
if the tests are passed, mean we don't have tests where `indexes[i] != i` yet.

Reply:
updated